### DISCUSSION TITLE
Business to Business and collecting payments then paying out

### DISCUSSION THREAD

**Discussion by StaffordInnovations** [flutter-stripe/flutter_stripe#868] [Q&A]
I wanted to know if Flutter_Stripe can support the business to business payments and collecting payments then paying out that Stripe can support?

https://stripe.com/docs/connect/enable-payment-acceptance-guide?platform=web
https://stripe.com/docs/connect/collect-then-transfer-guide 

in both cases they reference an application fee.

**Comment by lewisstevens1** [comment]
Currently, i cannot find any integration into stripe connect atm, in regards to the fee that will be the monthly fee per user as per this price. https://stripe.com/gb/connect/pricing (that monthly fee does not apply to the standard one)
